Model/src/main/java/Action/IdFactory.java

```diff
@@ -1,14 +1,17 @@
 package Action;
 
-import DataBase.DataBaseInf;
-import Users.Role;
+import java.util.concurrent.atomic.AtomicInteger;
 
-public class IdFactory {
-    private static int generalID =0;
-
-    public static int idBuilder () {
-        generalID ++;
-
-        return generalID;
+/**
+This class factory individual id for every
+essence in database
+**/
+public final class IdFactory {
+    private IdFactory() {
+    }
```

Comment:
Нужно добавить какой-нибудь комментарий
Фактически, это не фабрика, а генератор. 
```suggestion
        //to prevent instanse createion
    }
```

Reply:
Комментарий добавлен, генератор переделан.